.gitignore

```diff
@@ -1 +1,2 @@
-dist
+# dist
```

Comment:
I agree with explicitly ignoring `node_modules`, but we shouldn’t be checking in `dist`. Why is that change made in this PR?